autopr/models/rails.py

```diff
@@ -45,9 +45,7 @@ def get_rail_spec(cls):
 
 @xml_prefix_prompt
 
-{{output_schema}}
```

Comment:
`@xml_prefix_prompt` can also go. So you can substitute the older:

```
@xml_prefix_prompt

{{output_schema}}

@json_suffix_prompt_v2_wo_none
```

with a

```
@complete_json_suffix_v2
```